README.md

```diff
@@ -5,6 +5,28 @@
 
 ## Build Instructions
 
+### Condensed Steps
+
+Please refer to the sections below for more detailed information.
+
+1. Download and install Xcode.
+1. `git clone git@github.com:wordpress-mobile/WordPress-iOS.git` in the folder of your preference.
+1. `cd WordPress-iOS` to enter the working directory.
+1. `rake dependencies` this may take some time to complete.
+1. `rake xcode` to launch the workspace.
+1. Compile and run the app on a device or an emulator.
+
+In order to login to WordPress.com using the app:
+
+1. Create a WordPress.com account at https://wordpress.com/start/user (if you don't already have one).
+1. Create an application at https://developer.wordpress.com/apps/ 
+1. Set "Redirect URLs"= `https://localhost` and "Type" = `Native` and click "Create".
+1. Copy the `Client ID` and `Client Secret` from the OAuth Information. 
```

Comment:
After finishing the _Create_ step above, I get forwarded to this page. 

<img width="320" alt="app" src="https://user-images.githubusercontent.com/198826/57634788-3f8e1900-7563-11e9-8857-cff5f1fbf335.png">

It's not obvious where the `Client ID` and `Client Secret` should be copied from. Should we expand this or do you think it's understandable enough?

Reply:
I'll add a note in the previous step to click "Update".